src/main/java/com/sumi/smartemployeeservice/services/EmployeeService.java

```diff
@@ -0,0 +1,45 @@
+package com.sumi.smartemployeeservice.services;
+
+import com.sumi.smartemployeeservice.modal.Employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class EmployeeService {
+    @Autowired
+    private EmployeeRepository employeeRepository;
+
+
+    public List<Employee> getEmployees() {
+
+        List<Employee> employees = new ArrayList<>();
+        employeeRepository.findAll().forEach(employee -> employees.add(employee));
+        return employees;
```

Comment:
Why you need this?

Reply:
To get all employees from the database

Reply:
no. why you are creating new array list. it is waste of memory, you can just return empRepo.findAll()

Reply:
ooh. ok